lib/internal/test_runner/test.js

```diff
@@ -339,7 +326,7 @@ class Test extends AsyncResource {
         }
       } else {
         // This test is synchronous or using Promises.
-        await this.runInAsyncScope(this.fn, ctx, ctx);
+        await this.runInAsyncScope(this.fn, ctx, ...args);
```

Comment:
same here
```suggestion
        // Note that if it's not OK to mutate args, we need to first clone it.
        ArrayPrototypeUnshift(args, this.fn, ctx); // maybe worth to moved outside the if statement so it can be reused for the if case?
        const ret = ReflectApply(this.runInAsyncScope, this, args);
```